packages/mockyeah-cli/lib/__tests__/boot.test.js

```diff
@@ -0,0 +1,43 @@
+const { checkVersionMatchWithPackage } = require('../boot');
+
+describe('boot', () => {
+  describe('checkVersionMatchWithPackage', () => {
+    test('throws on missing CLI package version', () => {
+      expect(() => {
+        const env = {};
+        const pkgUp = {};
+        checkVersionMatchWithPackage(env, pkgUp);
+      }).toThrow('Could not find `mockyeah-cli` package version to check against core.');
+    });
+
+    test('throws on missing env package version', () => {
+      expect(() => {
+        const env = {};
+        const pkgUp = {
+          package: {
+            version: '0.0.0-test'
+          }
+        };
+        checkVersionMatchWithPackage(env, pkgUp);
+      }).toThrow('Could not find `mockyeah` package version to check against CLI.');
+    });
+
+    test('throws on mismatch package versions', () => {
+      expect(() => {
+        const env = {
+          modulePackage: {
+            version: '99.99.99-test'
+          }
+        };
+        const pkgUp = {
+          package: {
+            version: '0.0.0-test'
+          }
+        };
+        checkVersionMatchWithPackage(env, pkgUp);
+      }).toThrow(
+        'Version mismatch between CLI (0.0.0-test) and core (99.99.99-test) - please install same versions.'
+      );
+    });
+  });
```

Comment:
No positive test case?

Reply:
@ryanricard Excellent callout. We should look at wiring up code coverage here too somehow sometime. Added a positive test case in dc1ad34.